Makefile

```diff
@@ -2,14 +2,18 @@
 	clean \
 	deploy \
 	func-test \
-	unit-test
+	unit-test \
+	build
+
+SOURCES := $(shell find . -name '*.go' -not -path "*/vendor/*")
+IMG ?= quay.io/updateservice/updateservice-operator:latest
 
 clean:
 	@echo "Cleaning previous outputs"
 	rm functests/functests.test
 
 deploy:
-	@echo "Deploying Cincinnati operator"
+	@echo "Deploying Update Service operator"
 	hack/deploy.sh
 
 func-test: deploy
```

Comment:
A few lines down, you use `hack/functest.sh`, reverting #75.  Not sure why that doesn't show up in GitHub's diff, but:

```console
$ git diff origin/master..origin/pr/65 -- Makefile | grep functest
 	rm functests/functests.test
-	go test -v ./functests/...
+	hack/functest.sh
```

Reply:
I rebased against master and picked the change up and then carried it through.